services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+from db.models import User
+from django.db import transaction
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: User,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session_id=ticket["movie_session"],
+                order=order,
+            )
+
+
+def get_orders(username: str = None) -> QuerySet[Order]:
+    if username:
+        return Order.objects.filter(user__username=username)
+    else:
+        return Order.objects.all()
```

Comment:
You can simplify this code
```suggestion
    orders = Order.objects.all()
    if username:
        orders = orders.filter(user__username=username)
    return orders
```